app/assets/stylesheets/search.scss

```diff
@@ -0,0 +1,16 @@
+// Place all the styles related to the Search controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.search_form {
+
```

Comment:
I have some clean code habit. So if you can, please remove those useless spaces.